app/views/documentation/index.html.erb

```diff
@@ -42,8 +42,29 @@
     <p>When I'm done testing my changes, I can unlock the environment so others can deploy to it again.</p>
     <p><code>/deploy unlock staging on acme-inc/api</code></p>
     <hr />
-    <h5 id="auto-deployment">Auto Deployment</h5>
+    <h5 id="continuous-delivery">Continuous Delivery</h5>
     <p>You can configure SlashDeploy to automatically deploy a branch to an environment when you push code to GitHub or when a set of commit status contexts succeeds.</p>
+    <p>To configure CD, you add a <code>.slashdeploy.yml</code> file to the root of the repository. Here's an example config file that will automatically deploy the <code>master</code> branch to the <code>production</code> environment, once tests are passing on CircleCI:</p>
+    <p>
+      <pre>---
+production:
+  
+  # Allows us to reference this using "production" or a short alias, "prod".
+  aliases:
+    - prod
+
+  # Enable continuous delivery for this environment.
+  continuous_delivery:
+    # You can specify any git "ref" (tag, branch, etc) to auto deploy when
+    # pushed to.
+    ref: refs/heads/master
+
+    # If you want to only deploy when a set of GitHub commit statuses are
+    # passing, you can configure those here. If none are provided, SlashDeploy
+    # will deploy when new commits are pushed to the git ref.
+    required_contexts:
+      - ci/circleci</pre>
+    </p>
```

Comment:
Do you think it's worth talking about the SlashDeploy Github Integration `Single File` setting?

Reply:
This doc is more about usage docs. But I should definitely add this to the setup docs, if anybody wants to run SlashDeploy themselves.

Reply:
Agreed. I still think that is a decent goal, it's pretty cool.

Another reason to document it in the setup docs is that if SlashDeploy integration gets uninstalled by accident or something.